deepinv/tests/test_models.py

```diff
@@ -243,34 +242,37 @@ def __init__(self, model, *args, **kwargs):
             self.model = model
 
         def prox(self, x, w):
-            return self.model(x, w)
+            return self.model(x, w[:, 1, :, :].unsqueeze(1))
 
     class PDNetDataFid(DataFidelity):
         def __init__(self, model, *args, **kwargs):
             super().__init__(*args, **kwargs)
             self.model = model
 
         def prox(self, x, w, y):
-            return self.model(x, w, y)
+            return self.model(x, w[:, 2, :, :].unsqueeze(1), y)
```

Comment:
Same: shouldn't it be `w[:, 1, :, :].unsqueeze(1)`? Also, maybe use slicing: `w[:, 1:2, :, :]` without unsqueeze (or `w[:, 2:3, :, :]` in the case where you want channel index 2 and not 1)

Reply:
@samuro95 what are your thoughts?

Reply:
yes it is correct, I changed accordingly. Thanks Matthieu !